localstack-core/localstack/utils/crypto.py

```diff
@@ -184,3 +186,16 @@ def decrypt(
     decrypted = decryptor.update(encrypted) + decryptor.finalize()
     decrypted = unpad(decrypted)
     return decrypted
+
+
+def decrypt_rsa(key: bytes, encrypted: bytes) -> bytes:
+    private_key = serialization.load_der_private_key(key, password=None, backend=default_backend())
+    decrypted = private_key.decrypt(
+        encrypted,
+        padding.OAEP(
+            mgf=padding.MGF1(algorithm=hashes.SHA256()),
+            algorithm=hashes.SHA256(),
+            label=None,
+        ),
+    )
+    return decrypted
```

Comment:
Since this logic is centric to KMS I would recommend you to move this to the `kms/models.py` instead.